scripts/sync-and-test-bucket.sh

```diff
@@ -49,6 +49,9 @@ aws_region="$(pulumi -C infrastructure config get 'aws:region')"
 # fail, causing this script to exit nonzero. In either case, it's okay to continue.
 aws s3 mb $destination_bucket_uri --region $aws_region || true
 
+# Tag the bucket with ownership information
+aws s3api put-bucket-tagging --bucket $destination_bucket_uri --tagging file://bucket-tagging.json
```

Comment:
I think this should be relative to the project root:

```suggestion
aws s3api put-bucket-tagging --bucket $destination_bucket_uri --tagging "file://$(pwd)/scripts/bucket-tagging.json"
```

Reply:
Good idea on just having an absolute path